docs/pages/trustedclusters.mdx

```diff
@@ -274,7 +274,7 @@ spec:
       'environment': 'production'
 ```
 
-Now, we need to establish trust between roles "root:admin" and "leaf:admin". This is
+Now, we need to establish trust between roles "root:admin" and "leaf:local-admin". This is
```

Comment:
We should be moving away from the admin role and use access role as much as possible.